### PR TITLE
Create Toontown Cheatsheet

### PR DESCRIPTION
**Pull request by Startomas**
**What does your Instant Answer do?**
Gives mod commands for the Toontown Video Game

**What is the data source for your Instant Answer? (Provide a link if possible)**
Source code

**Why did you choose this data source?**
Most accurate 

**Are there any other alternative (better) data sources?**
Nope

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers, the Toontown community

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Are you having any problems? Do you need our help with anything?**
Nope!

**Where did you hear about DuckDuckHack? (For first time contributors)**
Duckduckgo search :smiley: !

-----
IA Page: https://duck.co/ia/view/toontown_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/toontown.json [hunk 1]
```diff
@@ -0,0 +1,382 @@
+{
+    "id": "toontown_cheat_sheet", 
+    "name": "Toontown",
+    "description": "Mod Commands", 
+	"aliases": [
+        "toontown mod", "toontown commands", "toontown mod commands", "tto mod commands", "tto commands"
+    ],
+
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/toontown.json]: Is there no source used that we can point to in the `meta` block? Is there any site we can send the user to for more details?

Comment (Startomas) [inline share/goodie/cheat_sheets/json/toontown.json]: There are no open source code aviable, nor I can make the one I used available. The only "source" are some forum posts.

The only other site we can link users to is the game's [wiki page](https://en.m.wikipedia.org/wiki/Toontown_Online)

File: share/goodie/cheat_sheets/json/toontown.json [hunk 2]
```diff
@@ -0,0 +1,386 @@
+{
+    "id": "toontown_cheat_sheet", 
+    "name": "Toontown Commands Cheet Sheet",
+    "description": "Commands For the Toontown Private Servers", 
+	"aliases": [
+        "toontown mod", "toontown commands", "toontown mod commands", "tto mod commands", "tto commands"
+    ],
+    "metadata" : {
++      "sourceName" : "Toontown Online Wiki",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/toontown.json]: these `+` signs will cause the parser to blow up; if you bomb them out them it'll start working again

Comment (Startomas) [inline share/goodie/cheat_sheets/json/toontown.json]: Never caught that, thanks!

**Comment by moollaza** [comment]
@Startomas I'm very sorry for the long delay here! This one fell through the crack. :frowning: 

It's now deployed to Beta: https://beta.duckduckgo.com/?q=toontown+cheat+sheet&ia=cheatsheet&iax=1

this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@Startomas congrats! It's live: https://duckduckgo.com/?q=Toontown+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**